java/apirag-core/src/test/resources/nutshop-c360.tools.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "type": "graphql",
+    "type": "api",
```

Comment:
Why are graphql APIs just called `api`?

Reply:
Because we now support graphql and rest apis via different executors. Hence, I moved what kind of API it is into the API configuration (in the main configuration file).
We now just need to distinguish between functions that are called through an API vs functions that are called locally vs functions that are passed to the frontend. Those are the 3 "types".